crates/sdk/test-utils/src/lib.rs

```diff
@@ -223,7 +223,7 @@ pub async fn create_and_sign_offchain_request(
     let job_params = JobParams {
         nonce,
         max_cycles,
-        // Need to use abi_encode_packed because the contract address
+        // Need to use ivm_abi_encode_packed because the contract address
```

Comment:
I'm guessing this got changed by mistake lol
```suggestion
        // Need to use abi_encode_packed because the contract address
```